cmd/crio/main.go

```diff
@@ -306,6 +309,11 @@ func main() {
 			Value: "error",
 			Usage: "log messages above specified level: debug, info, warn, error (default), fatal or panic",
 		},
+		cli.StringFlag{
+			Name:  "log-dir",
+			Value: "",
+			Usage: "default log directory were all logs will go unless directly specified by the kubelet",
```

Comment:
OK, now you've a change.  This is customer facing.  s/were/where/

Reply:
Fixed :)